manifests/server/tls/ca.pp

```diff
@@ -17,7 +17,7 @@
     Package <| title == 'openssl' |>
     -> exec {
       default:
-        path    => ['/usr/bin', '/bin'];
+        path => pick($facts['path'], '/usr/bin:/bin');
```

Comment:
I don't think `pick()` is required here. Is there actually a scenario where `PATH` could not exist?

Reply:
I have this niggling reminder of that particular fact being invalid on some system at us, but I'm not entirely sure if that's true any more.